CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+## [1.7.0] - 2022-09-14
+
+### Added
```

Comment:
We are missing some issues in here. The combined building damage analysis for wind, wave and surge is missing. Maybe check the milestone issues for any others that are missing.